deploy/manifests/dev/us-east-2/cluster/external-dns/deploy-patch.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-dns
+  namespace: external-dns
+spec:
+  selector:
+    matchLabels:
+      app: external-dns
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      containers:
+        - name: external-dns
+          args:
+            - --source=service
+            - --source=ingress
+            - --domain-filter=dev.cid.contact
+            - --provider=aws
+            - --policy=upsert-only
+            - --aws-zone-type=public
+            - --registry=txt
+            - --txt-owner-id=Z0346653G546POP574CS
```

Comment:
Where does this come from?

Reply:
[Here](https://github.com/filecoin-project/storetheindex/blob/2bb47c38c3ca0200e4da9605a1c7c3f8b03399cc/deploy/infrastructure/dev/us-east-2/route53.tf).

Reply:
From a terraform output?
Maybe a comment that says something to that effect? Something like
> Value is from `terraform output <name>`